__device-tests__/pages/editor-page.js

```diff
@@ -46,7 +46,7 @@ export default class EditorPage {
 	// Finds the wd element for new block that was added and sets the element attribute
 	// and accessibilityId attributes on this object and selects the block
 	// position uses one based numbering
-	async getBlockAtPosition( position: number, blockName: string ) {
+	async getBlockAtPosition( position: number, blockName: string = '' ) {
```

Comment:
Hmm I'm not seeing where you used this default value, is there a reason you included it here?